src/plugins/intel_gpu/src/graph/impls/implementation_map.hpp

```diff
@@ -147,7 +147,7 @@ class implementation_map {
                 return factory;
             }
         }
-        throw std::runtime_error(std::string("implementation_map for ") + typeid(primitive_kind).name() +
+        throw std::runtime_error(std::string("implementation_map for ") + primitive.id() +
                                      " could not find any implementation to match key");
```

Comment:
What do you think about extending error message so that it contains information about requested key as well? Something like: `... could not find any implementation to match key <bfyx, f32>`